optimade/server/entry_collections/mongo.py

```diff
@@ -21,11 +19,13 @@
 
 
 if CONFIG.use_real_mongo or CI_FORCE_MONGO:
+    from pymongo.collection import Collection as MongoCollection
```

Comment:
This clashes with the name of the OPTIMADE `EntryCollection` for Mongo

Reply:
Ah of course. I just wanted something more meaningful. I could just import `collection` instead, or `Collection` and not rename is?

Reply:
I thought about moving it to a completely different file as well, which handles instantiating the DB... but there shouldn't be a need for this in retrospect.

Reply:
or `DriverCollection`? 😅 ... will just change it to not renaming the import...

Reply:
I'd hold off on any extra changes until I have time to finish off #660, you can either suggest them in a review there or get them in yourself afterwards :)